tools/travis/build.sh

```diff
@@ -38,6 +38,11 @@ cat $WHISKDIR/whisk.properties
 cd $ROOTDIR/packages
 ./installCatalog.sh $WHISKDIR/config/keys/auth.whisk.system
 
+# Set credentials
+VCAP_SERVICES_FILE="$(readlink -f $ROOTDIR/tests/credentials.json)"
```

Comment:
I did not see the reference to  the repo https://github.com/openwhisk/openwhisk-package-template.

I only see credentials.json added here. Could you give more explanations?